src/System.ComponentModel.Composition/tests/IExportableTest.cs

```diff
@@ -0,0 +1,12 @@
+using System;
```

Comment:
Many of your files are missing the license. To check, do something liek this:
`for /f %i in ('dir /s/b *') do @(@echo %i & head -4 %i)`

I got `head` etc from the GNU tools for Win32 or you can use bash.

Always look for ways to get a simple script to do the hard work for you :)

Reply:
I'll make sure to address this as well. thanks